### PR TITLE
feat: add bulk subscriber api

### LINKED ISSUE #31: [NV-GO-5] API 🚀: Add Bulk Subscriber Creation Feature
The bulk subscription feature just became available. This SDK needs to be updated to have this feature.

API reference: https://docs.novu.co/api-reference/subscribers/bulk-create-subscribers

### PR DESCRIPTION
**Pull request by mahendraHegde**
closes #31

### Changes
- add sdk method to to bulk subscribe
- noticed jsob tag missmatch with SubscriberPayload fixed
- add relavant test cases

### CODE REVIEW

**Comment by mahendraHegde** [comment]
@sumitsaurabh927  @unicodeveloper  can review plz?

File: lib/subscribers.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 
 type ISubscribers interface {
 	Identify(ctx context.Context, subscriberID string, data interface{}) (SubscriberResponse, error)
+	CreateBulk(ctx context.Context, subscribers SubscriberBulkPayload) (SubscriberBulkCreateResponse, error)
```
Comment (unicodeveloper) [inline lib/subscribers.go:16]: Please rename this to `BulkCreate`

File: lib/subscribers.go [hunk 2]
```diff
@@ -49,6 +50,26 @@ func (s *SubscriberService) Identify(ctx context.Context, subscriberID string, d
 	return resp, nil
 }
 
+func (s *SubscriberService) CreateBulk(ctx context.Context, subscribers SubscriberBulkPayload) (SubscriberBulkCreateResponse, error) {
```
Comment (unicodeveloper) [inline lib/subscribers.go:53]: => `BulkCreate`

**Review by unicodeveloper — COMMENTED** [review]
Looks good. Please attend to the comments.

File: lib/subscribers.go (hunk 2 above)
Comment (mahendraHegde) [reply lib/subscribers.go]: done

File: lib/subscribers.go (hunk 1 above)
Comment (mahendraHegde) [reply lib/subscribers.go]: done

**Review by sumitsaurabh927 — APPROVED** [review]
Great work @mahendraHegde 🎉

Check out our other issues for more Hacktoberfest contributions and jump into our community to interact with other fellow contributors, and our team members and to get help.

Here's the [link](https://discord.gg/novu)

**Pull request merged**